internal/cli/atlas/backup/schedule/update.go

```diff
@@ -90,13 +102,75 @@ func (opts *UpdateOpts) NewBackupConfig(cmd *cobra.Command, clusterName string)
 		out.RestoreWindowDays = &opts.restoreWindowDays
 	}
 
-	out.AutoExportEnabled = returnValueForSetting(opts.autoExport, opts.noAutoExport)
-	out.UpdateSnapshots = returnValueForSetting(opts.updateSnapshots, opts.noUpdateSnapshots)
-	out.UseOrgAndGroupNamesInExportPrefix = returnValueForSetting(opts.useOrgAndGroupNamesInExportPrefix, opts.noUseOrgAndGroupNamesInExportPrefix)
+	out.AutoExportEnabled = cli.ReturnValueForSetting(opts.autoExport, opts.noAutoExport)
+	out.UpdateSnapshots = cli.ReturnValueForSetting(opts.updateSnapshots, opts.noUpdateSnapshots)
+	out.UseOrgAndGroupNamesInExportPrefix = cli.ReturnValueForSetting(opts.useOrgAndGroupNamesInExportPrefix, opts.noUseOrgAndGroupNamesInExportPrefix)
+
+	if cmd.Flags().Changed(flag.BackupPolicy) {
+		currentSchedule, err := opts.store.DescribeSchedule(opts.ConfigProjectID(), opts.clusterName)
+		if err != nil {
+			return nil, err
+		}
+		policies := currentSchedule.Policies
+
+		for _, backupPolicy := range opts.backupPolicy {
+			policyItems := strings.Split(backupPolicy, ",")
+			frequencyInterval, err := strconv.Atoi(policyItems[3])
+			if err != nil {
+				return nil, err
+			}
+			retentionValue, err := strconv.Atoi(policyItems[5])
+			if err != nil {
+				return nil, err
+			}
+
+			policyIndex := findPolicyIndex(policyItems[0], policies)
+
+			if policyIndex == -1 {
+				return nil, errors.New("incorrect value for parameter policyID. Policy with such ID does not exist")
```

Comment:
can you add new policies with this endpoint?

Reply:
@fmenezes No, I don't think it's possible. If we try to add a new item with an unknown ID, the backend returns:
```
{
    "detail": "The specified policy item id 62da8faac84a2721e448d76c does not exist.",
    "error": 400,
    "errorCode": "INVALID_POLICY_ITEM_ID",
    "parameters": [
        "62da8faac84a2721e448d76c"
    ],
    "reason": "Bad Request"
}
```